dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateSQL.java

```diff
@@ -0,0 +1,63 @@
+package com.dotmarketing.portlets.templates.business;
+
+import com.dotmarketing.beans.Inode.Type;
+
+public class TemplateSQL {
```

Comment:
Codacy found an issue: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6380036)